svg/elements/path.json

```
@@ -94,6 +94,67 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "path": {
+            "__compat": {
+              "description": "<code>path()</code> function can be used to set value in CSS",
```

Does this mean something different than "you can use `path()` in the `d` attribute"? If not…

```suggestion
              "description": "<code>path()</code>",
```

If I've misunderstood, then help me out here because I'm not sure what distinction the current description text is making. Maybe give me an example?

---

@ddbeck In summary, I added that description because everything else is about "static" SVG. The uninformed seeing `path()` might assume it is something that you specify in an SVG file. Actually it is something you specify in CSS to modify the SVG - as shown in https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d#using_d_as_a_css_property

I've now added an MDN URL to the function to which will help here. 

Upshot, if you think the distinction is something for MDN rather than BCD I'm OK with that - either add a further response or press that commit button!

---

This is still an outstanding question, or it got addressed elsewhere and it got lost in the shuffle. An answer or a pointer in the right direction would be helpful here.

---

@ddbeck I "definitely" wrote an answer for this - but as you say, not present.

My thinking was that everything else about this definition indicates support for SVG compatibility - so `svg.elements.path.d` indicates that you can set the SVG path in the `d` element. Now we have added `svg.elements.path.d.path` which is a function that allows you to set this SVG value in CSS. 

That seemed really unexpected to me: enough that clarifying with a proper description seemed sensible. Otherwise you might read this as SVG could call the path() function. 

Of course for BCD you have to read the docs to understand what the compatibility means, so this is not an unreasonable suggestion.

SO. Please self-commit and merge if you still think this should happen, or just merge if you think my argument is compelling :-)

---

That is correct! I'll commit that.